akka-docs/src/main/paradox/stream/operators/Source/zipN.md

```diff
@@ -1,20 +1,40 @@
 # Source.zipN
 
-Combine the elements of multiple streams into a stream of sequences.
+Combine the elements of multiple sources into a source of sequences of value.
 
 @ref[Source operators](../index.md#source-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #zipN }
-
-@@@
+@apidoc[Source.zipN](Source$) { scala="#zipN[T](sources:scala.collection.immutable.Seq[akka.stream.scaladsl.Source[T,_]]):akka.stream.scaladsl.Source[scala.collection.immutable.Seq[T],akka.NotUsed]" java="#zipN(java.util.List)" } 
 
 ## Description
 
-Combine the elements of multiple streams into a stream of sequences.
+Collects one element for every upstream and when all upstreams has emitted one element all of them are emitted downstream as a collection.
+The element order in the downstream collection will be according to the order their respective sources was listed.
```

Comment:
Optional: Took me a while to grok this sentence, maybe will be the same order as the sources were listed

Reply:
changed!